packages/fluentui/react-builder/src/components/Toolbar.tsx

```diff
@@ -41,68 +41,138 @@ export const Toolbar: React.FunctionComponent<ToolbarProps> = ({
+              onClick: () => {
+                onModeChange('build');
+              },
+            },
+            {
+              key: 'design',
+              content: 'Design',
+              onClick: () => {
+                onModeChange('design');
+              },
+            },
+            {
+              key: 'use',
+              content: 'Use',
+              onClick: () => {
+                onModeChange('use');
+              },
+            },
+          ]}
+          on="click"
+        />
+      </div>
+      <FUIToolbar
+        aria-label="Builder toolbar"
         items={[
           {
-            key: 'build',
-            label: 'Build',
-            value: 'build',
+            key: 'divider-1',
+            kind: 'divider',
+          },
+          {
+            icon: (
+              <UndoIcon
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'undo',
+            // kind: 'toggle',
+            disabled: !canUndo,
+            title: 'Undo',
+            onClick: () => {
+              onUndo;
+            },
           },
           {
-            key: 'design',
-            label: 'Design',
-            value: 'design',
+            icon: (
+              <RedoIcon
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'redo',
+            // kind: 'toggle',
+            disabled: !canRedo,
+            title: 'Redo',
+            onClick: () => {
+              onRedo;
+            },
           },
           {
-            key: 'use',
-            label: 'Use',
-            value: 'use',
+            key: 'divider-2',
+            kind: 'divider',
+          },
+          {
+            icon: (
+              <TrashCanIcon
+                {...{
+                  outline: true,
+                }}
+              />
+            ),
+            key: 'delete',
+            // kind: 'toggle',
+            title: 'Start over',
+            onClick: () => {
+              onReset;
+            },
           },
         ]}
       />
     </div>
-    &nbsp; &nbsp;
-    <Button
-      text
-      icon={<OpenOutsideIcon />}
-      content="Popout"
-      onClick={() => {
-        window.open(`/builder/maximize${window.location.hash}`, '_blank', 'noopener noreferrer');
-      }}
-    />
-    <Button text icon={<UndoIcon />} content="Undo" onClick={onUndo} disabled={!canUndo} />
-    <Button text icon={<RedoIcon />} content="Redo" onClick={onRedo} disabled={!canRedo} />
     <div style={{ display: 'flex', alignItems: 'center', marginLeft: 'auto' }}>
-      <Checkbox label="Show Code" toggle checked={!!showCode} onChange={(e, data) => onShowCodeChange(data.checked)} />
-      &emsp;
-      <Checkbox
-        label="Show JSON"
-        toggle
-        checked={!!showJSONTree}
-        onChange={(e, data) => onShowJSONTreeChange(data.checked)}
+      <MenuButton
```

Comment:
Does MenuButton menu support a single select option that would show a checkmark next to the text when selected?

Reply:
Example seen here, where it would be nice to only allow one item to be checked at a time (since only code OR json is shown below, but not ever both). Also, to show a checkmark next to the selected option.

Reply:
no, this is not supported out of the box for menubutton yet, but you can turn that whole group into a toolbar and use https://fluentsite.z22.web.core.windows.net/0.51.3/components/toolbar/definition?showCode=false&showRtl=false&showTransparent=false&showVariables=false#content-menu-radio-group

Reply:
alternatively, you can use `<Dropdown />` but that looks a bit differently

Reply:
I'm going to keep as-is for now. Will update in a future PR or wait for support in MenuButton